pkg/urbit/jets/e/cue.c

```diff
@@ -3,94 +3,204 @@
 */
 #include "all.h"
 
+typedef struct cueframe
```

Comment:
Docs on these fields would be helpful. I can guess about most of them, but `wid` and `hed` especially could use some explanation.